test/controllers/sessions_controller_test.rb

```diff
@@ -31,13 +31,4 @@ class SessionsControllerTest < ActionController::TestCase
       end
     end
   end
```

Comment:
Removed test 'the ldap server has cant find the user data' will be updating this test by removing VCR and using Webmock Stubs and fixtures, see card https://trello.com/c/zThoJQ4Z